/**:
  ros__parameters:
    initial_pose: [0.0, 0.0, 0.0]